### ISSUE TITLE
UUIDs aren't included in the config by default

### ISSUE THREAD

**Issue by Arena-Sniper** [nvb-uy/CIA#9]
UUIDs are how Minecraft determines the IDs of attributes, if they have the same ID they will conflict with each other and only use the first attribute, this is likely how https://github.com/nvb-uy/CIA/issues/1 happened.

**Comment by Arena-Sniper** [comment]
The solution is to add a UUID option, so players don't get conflicting attributes

**Comment by nvb-uy** [comment]
I will do this soon ^^

**Comment by Pandaismyname1** [comment]
A simple fix would be to add the item name into the UUID generation, no extra config needed.
```Java
EntityAttribute attribute = generic_attribute.getAttribute();
Collection<EntityAttributeModifier> newModifiers = new ArrayList<>();
var identifierString = stack.toString() + generic_attribute.getString();

newModifiers.add(
        new EntityAttributeModifier(
                UUID.nameUUIDFromBytes(identifierString.getBytes()),
                "Custom Item Attributes Modifier",
                generic_attribute.getDouble(),
                generic_attribute.getOperation()
        ));
```

**Comment by nvb-uy** [comment]
hm yeah that's a good idea! will implement this on the next version :)

**Comment by nvb-uy** [comment]
Fixed on 2.0.0, please read full changelog as the config moved locations and the format also changed.
